### PR TITLE
Match stage_unknown_sprite

### PR DESCRIPTION
**Pull request by JaceCear**

### CODE REVIEW

File: src/game/stage_unknown_sprite.c [hunk 1]
```diff
@@ -0,0 +1,85 @@
...
+        = TaskCreate(sub_800A5F8, sizeof(Sprite_StageSprUnknown), taskPrio, 0, dtor);
+
+    Sprite_StageSprUnknown *su = TaskGetStructPtr(t);
+    su->unk30 = 0;
+    su->unk34 = 0;
+    su->unk38 = 0;
+    su->unk3A = 0;
+    su->unk3C = 0;
+    su->unk3E = 0;
+    su->unk40 = 0;
```
Comment (freshollie) [inline src/game/stage_unknown_sprite.c:35]:
```suggestion
    su->unk30 = 0; // x
    su->unk34 = 0; // y
    su->unk38 = 0; // speedX
    su->unk3A = 0; // speedY
    su->unk3C = 0; // accX
    su->unk3E = 0; // accY
    su->unk40 = 0;
```

Thoughts?

Comment (JaceCear) [reply src/game/stage_unknown_sprite.c:35]: Sounds good!

**Review by freshollie — APPROVED** [review]

**Comment by freshollie** [comment]
@JaceCear I'm confused, are you merging this?

File: include/game/enemies/mon.h [hunk 2]
```diff
@@ -0,0 +1,19 @@
+#ifndef GUARD_ENEMY_MON_H
```
Comment (freshollie) [inline include/game/enemies/mon.h:1]: Ah sorry, missed it!

File: include/game/enemies/mon.h [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+typedef struct {
+    /* 0x00 */ SpriteBase base;
+    /* 0x0C */ Sprite s;
+    /* 0x3C */ s32 x;
+    /* 0x40 */ s32 y;
+    /* 0x44 */ s32 deltaY;
+    /* 0x48 */ s32 speedY;
+} Sprite_Mon; /* 0x4C */
+
+void initSprite_Enemy_Mon(MapEntity *me, u16 spriteRegionX, u16 spriteRegionY,
```
Comment (freshollie) [inline include/game/enemies/mon.h:16]: Function is wrong

File: include/game/enemies/mon.h [hunk 4]
```diff
@@ -0,0 +1,19 @@
+#ifndef GUARD_ENEMY_MON_H
+#define GUARD_ENEMY_MON_H
+
+#include "global.h"
+#include "game/entity.h"
+
+typedef struct {
```
Comment (freshollie) [inline include/game/enemies/mon.h:7]: And struct doesn't need to be in here, it's effectively private

File: include/game/enemies/mon.h (hunk 3 above)
Comment (JaceCear) [reply include/game/enemies/mon.h:16]: Well, it's also wrong in Bell then! :D 
I'll have a look at the other headers.

**Pull request merged**